src/Standards/Generic/Tests/NamingConventions/UpperCaseConstantNameUnitTest.1.inc

```diff
@@ -67,3 +67,16 @@ class MyClass {}
     define('some param')
 ]
 class MyClass {}
+
+const MixedCase = 1;
+
+define('lower_case_name', 'value');
+define($var, 'sniff should bow out');
+define(constantName(), 'sniff should bow out');
+define($obj->constantName(), 'sniff should bow out');
+define(MyClass::constantName(), 'sniff should bow out');
+define(condition() ? 'name1' : 'name2', 'sniff should bow out');
+
+// Valid if outside the global namespace. Sniff should bow out.
+function define($param) {}
```

Comment:
> Maybe this could also use a code sample with a method called define() in a class ?

Done

> As for the code sample here, would it be clearer if the code was wrapped in a (scoped) namespace ? Or even just had a namespace declaration above it ?
> Not that it would make a difference for the sniff, but just wondering. No change is needed as the comment is valid as is.

I ended up not changing the test as you mentioned no change was needed, and I was a bit torn about which option I prefer.